.github/workflows/continuous-integration-workflow.yml

```diff
@@ -10,9 +10,9 @@ on:
     branches:
       - "**"
 
-jobs:  
+jobs:
   test:
-    name: Test Suite
+    name: Test on GitHub Runners
```

Comment:
Random question as I may be missing something, but why is this kept? Is it because the self-hosted runners aren't on linux?

Reply:
Hm no they are on Linux... I guess it just felt more complete to keep it, in case something goes wrong with the self-hosted runners? But I'm fine with axing it if you think that makes more sense

Reply:
I'm not really opinionated on the matter, mostly asked out of curiosity. Removing it would also allow for much faster CI jobs completion in general, so quite convenient, but your point is valid.

Reply:
Good point... @dlubarov thoughts?

Reply:
I don't have the full context but if these are beefier VMs, I agree it would be nice to remove the old config for the speedup.